docs/guides/ucan-server.md

```diff
@@ -0,0 +1,182 @@
+4. ✅ Access to a PDP server (Piri or Curio) at an HTTPS endpoint
+5. ✅ [Configured TLS termination](../common/tls-termination.md) for your domain
+
+### PDP Backend Requirement
+
+You need ONE of the following:
+- **Piri PDP Server**: Follow the [PDP server guide](./pdp-server-piri.md)
+- **Curio**: See [Curio integration guide](../integrations/piri-with-curio.md)
+
+Your PDP backend must be accessible via HTTPS (e.g., `https://pdp.example.com`).
+
+## Step 1: Prepare Your Identity
+
+### Derive Your DID
+
+Using the PEM file you generated:
+
+```bash
+# Extract your DID
+piri id parse service.pem | jq .did
+```
+
+Example output: `did:key:z6MkhaXgD8CkBJPgWK2mPx7kW6KnvgCx6LJfnv4jkLd4VWQR`
+
+Save this DID - you'll need it for Storacha registration.
+
+### For Curio Integration
+
+If using Curio as your PDP backend:
+
+1. Extract the public key from your PEM:
+   ```bash
+   openssl pkey -in service.pem -pubout
+   ```
+
+2. Add this public key to Curio's configuration as a service named `storacha`
+3. See the [Curio integration guide](../integrations/piri-with-curio.md) for details
+
+## Step 2: Create a Proof Set
+
+Before accepting data, you need to create a proof set on the PDP backend:
+
+```bash
+piri proofset create \
+  --key-file=service.pem \
+  --pdp-server-url=https://YOUR_PDP_DOMAIN \
+  --record-keeper=0x6170dE2b09b404776197485F3dc6c968Ef948505
+```
+
+Note: The [record-keeper](https://github.com/FilOzone/pdp/?tab=readme-ov-file#contracts) address is the contract address on the Calibration network.
+
+### Monitor Creation
+
+Check the status of your proof set creation:
+
+```bash
+piri proofset status \
+  --key-file=service.pem \
+  --curio-url=https://YOUR_PDP_DOMAIN \
+  --ref-url=/pdp/proof-set/created/HASH_FROM_CREATE_OUTPUT
+```
+
+Wait until you see:
+```json
+{
+  "proofsetCreated": true,
+  "txStatus": "confirmed",
+  "ok": true,
+  "proofSetId": YOUR_PROOFSET_ID
+}
+```
+
+Save your `proofSetId` for reference.
+
+## Step 3: Register with Storacha Delegator
+
+### Registration Process
+
+1. **Start Piri temporarily** to prepare for delegation:
+   ```bash
+   piri start --key-file=service.pem
+   ```
+
+2. **Share your DID** with the Storacha team via appropriate channels
+
+3. **Visit the Delegator**:
+   - https://staging.delegator.storacha.network
+
+4. **Follow instructions** to:
+   - Receive your delegation proof
+   - Generate a delegation proof
+   - Environment configuration
+
+5. **Stop the temporary server** (Ctrl+C) after receiving your configuration
+
+### Save Configuration
+
+Create a `.env` file with the configuration received:
+
+```bash
+# Example .env file
+INDEXING_SERVICE_DID="did:web:staging.indexing.storacha.network"
+INDEXING_SERVICE_URL="https://staging.indexing.storacha.network"
+INDEXING_SERVICE_PROOF="bafyrei..."
+UPLOAD_SERVICE_DID="did:web:staging.upload.storacha.network"
+UPLOAD_SERVICE_URL="https://staging.upload.storacha.network"
```

Comment:
Can the build encode these as defaults?

Reply:
I've been using https://github.com/storacha/piri/blob/main/pkg/presets/presets.go.
We can change this, but I think that may break the AWS lambda deployment?
Will revisit in https://github.com/storacha/piri/issues/114 and possibly a follow on to https://github.com/storacha/piri/issues/108